back-end/lms/models/student.py

```diff
@@ -32,7 +32,56 @@ class Student(Person):
     birthdate = models.DateField()
     program = models.ForeignKey(Program, models.DO_NOTHING)
     status = models.ForeignKey(Status, models.DO_NOTHING)
-    photo = models.CharField(max_length=128, blank=True, null=True)
+    photo = models.ImageField(upload_to = '', max_length=128)
+    place_birth = models.TextField()
+    region = models.TextField()
+    adress = models.TextField()
+    pasport = models.TextField() #можно числовой, но проще так
+    pasport_ufms = models.TextField()
+    pasport_ufms_code = models.TextField()
+    pasport_date = models.DateField()
+    citizence = models.TextField()
+    fio_rod = models.TextField()
+    hse_id = models.TextField()
+    faculty = models.TextField()
+    edu_program = models.TextField()
+    hse_group = models.TextField()
+    #mother
+    mother_fio = models.TextField()
```

Comment:
NEVER use combined name fields. Store _name_, _family name_ and _patronymic_ separately. With special view functions, it is possible to combine these 3 fields to create the necessary _full name_ field, if you need that